src/components/BoardCard/styles.scss

```diff
@@ -11,41 +11,50 @@
   margin: 15px;
 
   animation: fadein 1s;
+
```

Comment:
Not a huge deal but one tip for the future is minimizing whitespace diffs to keep your pull requests shorter and easily readable